public/404.html

```diff
@@ -58,10 +62,14 @@
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
+      <h2>Publify</h2>
+      <h1>404 Not Found</h1>
```

Comment:
What about having the `<h2>` and `<h1>` the other way around, `<h1>` then `<h2>`, to go semantically with heading hierarchy?